build/version.py

```diff
@@ -57,6 +57,8 @@ def extractRevision():
 	if not isdir('derived'):
 		makedirs('derived')
 	log = open('derived/version.log', 'w')
+	print('Extracting version info...', file = log)
+	print('Package version: %s' % packageVersion, file = log)
```

Comment:
Perhaps we can bring this file (even) more in sync with the openMSX version in build/version.py?